### PR TITLE
Implement stub for high-scores

### PR DESCRIPTION
**Pull request by kytrinyx**
Here are a couple of reference implementations from other languages:
- https://github.com/exercism/csharp/blob/main/exercises/practice/high-scores/.meta/Example.cs
- https://github.com/exercism/fsharp/blob/main/exercises/practice/high-scores/.meta/Example.fs
- https://github.com/exercism/ruby/blob/main/exercises/practice/high-scores/.meta/example.rb

### CODE REVIEW

**Comment by lucaferranti** [comment]
did something happen while generating the testsuite? Based on the instructions, I understand one should only implement three functions (top, topThree, latest), but the test suite has also other ones

**Comment by kytrinyx** [comment]
I wonder if I need to update the instructions again? I'll check.

**Comment by kytrinyx** [comment]
The upstream specification has these tests, but no mention of the functions in the description.

I think my preference would be to regenerate the test suite without the extra tests so that the test suite is consistent with the instructions.

**Comment by kytrinyx** [comment]
I regenerated the test suite without all the extra functions.

File: exercises/practice/high-scores/.meta/tests.toml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+[1035eb93-2208-4c22-bab8-fef06769a73c]
+description = "List of scores"
+include = false
+
+[6aa5dbf5-78fa-4375-b22c-ffaa989732d2]
+description = "Latest score"
+
+[b661a2e1-aebf-4f50-9139-0fb817dd12c6]
+description = "Personal best"
+include = false
```
Comment (lucaferranti) [inline exercises/practice/high-scores/.meta/tests.toml]: I think we could include this?

Comment (kytrinyx) [reply exercises/practice/high-scores/.meta/tests.toml]: Yepp. I was going off of which methods were described in the instructions:

> Your task is to write methods that return the highest score from the list, the last added score and the three highest scores.

If you don't think it's confusing, let's include it. (I'll regenerate)

**Comment by kytrinyx** [comment]
This looks great!

**Review by lucaferranti — APPROVED** [review]

**Pull request merged**